pinot-common/src/main/java/org/apache/pinot/common/utils/TlsUtils.java

```diff
@@ -255,6 +262,38 @@ public static URL makeKeyStoreUrl(String storePath)
     return inputUri.toURL();
   }
 
+  public static SSLContext getSslContext() {
+    return SSLContextHolder.SSL_CONTEXT;
+  }
+
+  public static void setSslContext(SSLContext sslContext) {
+    registerOverride(sslContext);
+  }
+
+  private static void registerOverride(SSLContext override) {
+    if (!SSL_CONTEXT_REF.compareAndSet(null, override)) {
+      LOGGER.warn("SSL Context has already been set.");
+      // warn that something else beat the caller here
+    }
+  }
```

Comment:
oh I'd just inline this into `setSslContext`, `registerOverride` was just a name